app/scripts/directives/createSecret.js

```diff
@@ -117,38 +117,38 @@ angular.module("openshiftConsole")
             data: {}
```

Comment:
nit: It's a little simpler to change `data` here to `stringData` and just set `secret.stringData` in each case block below.